### PR TITLE
Add release-4.5 branch config for powervm

### PR DESCRIPTION
**Pull request by yussufsh**
Signed-off-by: Yussuf Shaikh <yussuf.shaikh@ibm.com>

### CODE REVIEW

**Comment by ltccci** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/ppc64le-cloud/test-infra/pull/68#" title="Author self-approved">yussufsh</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=ppc64le-cloud%2Ftest-infra).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[config/jobs/ocp-automation/OWNERS](https://github.com/ppc64le-cloud/test-infra/blob/master/config/jobs/ocp-automation/OWNERS)~~ [yussufsh]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: config/jobs/ocp-automation/ocp4-upi-powervm/ocp4-upi-powervm-presubmit.yaml
Comment (mkumatag) [inline config/jobs/ocp-automation/ocp4-upi-powervm/ocp4-upi-powervm-presubmit.yaml:8]: you don't need this because this repo is public

File: config/jobs/ocp-automation/ocp4-upi-powervm/ocp4-upi-powervm-presubmit.yaml
Comment (mkumatag) [inline config/jobs/ocp-automation/ocp4-upi-powervm/ocp4-upi-powervm-presubmit.yaml:30]: same as above

**Review by mkumatag — CHANGES_REQUESTED** [review]
minor comments

File: config/jobs/ocp-automation/ocp4-upi-powervm/ocp4-upi-powervm-presubmit.yaml
Comment (yussufsh) [reply config/jobs/ocp-automation/ocp4-upi-powervm/ocp4-upi-powervm-presubmit.yaml:8]: oki then also remove it from powervs configs?

File: config/jobs/ocp-automation/ocp4-upi-powervm/ocp4-upi-powervm-presubmit.yaml
Comment (yussufsh) [reply config/jobs/ocp-automation/ocp4-upi-powervm/ocp4-upi-powervm-presubmit.yaml:30]: Will remove.

File: config/jobs/ocp-automation/ocp4-upi-powervm/ocp4-upi-powervm-presubmit.yaml
Comment (mkumatag) [reply config/jobs/ocp-automation/ocp4-upi-powervm/ocp4-upi-powervm-presubmit.yaml]: yes, if that is public :)

**Comment by ltccci** [comment]
@yussufsh: Updated the `job-config` configmap in namespace `prow` at cluster `default` using the following files:
 - key `ocp4-upi-powervm-presubmit.yaml` using file `config/jobs/ocp-automation/ocp4-upi-powervm/ocp4-upi-powervm-presubmit.yaml`

<details>

In response to [this](https://github.com/ppc64le-cloud/test-infra/pull/68):

>Signed-off-by: Yussuf Shaikh <yussuf.shaikh@ibm.com>


Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

**Pull request merged**
